cd/cd_dump.ixx

```diff
@@ -917,8 +967,12 @@ export bool redumper_dump_cd(Context &ctx, const Options &options, bool refine)
         {
             if(lba == lba_refine)
             {
-                LOGC_RF("{} [{:3}%] LBA: {:6}/{}, errors: {{ SCSI: {}, C2: {}, Q: {} }}", spinner_animation(),
-                    percentage(refine_processed * refine_retries + refine_counter, refine_count * refine_retries), lba, lba_overread, errors_scsi, errors_c2, errors_q);
+                std::string rw_log;
+                if(options.check_subchannel_rw)
+                    rw_log = std::format(", RW: {}", errors_rw);
```

Comment:
To avoid confusion with CD-RW, "RW" should be consistently be written "R-W" or maybe "R..W" in user-facing text.